lib/calendar.ts

```diff
@@ -1537,17 +1459,18 @@ const helperHebrew: HelperSharedImpl = ObjectAssign({}, helperSharedImpl, {
       if (monthExtra) {
         const monthInfo = this.months[monthExtra];
         if (!monthInfo) throw new RangeError(`Unrecognized month from formatToParts: ${monthExtra}`);
-        month = this.inLeapYear({ year }) ? monthInfo.leap : monthInfo.regular;
+        month = (this.inLeapYear({ year }) ? monthInfo.leap : monthInfo.regular) as number;
```

Comment:
Does the `as number` work because the `inLeapYear` result makes sure that the `undefined` variant is never selected?